grails-app/migrations/0.8.x/changelog-2022-09-02-1200-add-default-product-type.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <property name="now" value="now()"/>
+
+  <changeSet author="awalkowiak" id="0209202212000-0">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="0">SELECT COUNT(*) FROM product_type WHERE id like "DEFAULT" OR code like "DEFAULT" OR name like "DEFAULT"</sqlCheck>
```

Comment:
What's the difference between like and equality here?

Reply:
Is like just case-insensitive equals?

Reply:
Yup, I wanted to take whatever default product type someone already created